SoftLayer/CLI/event_log/get.py

```diff
@@ -56,8 +57,11 @@ def cli(env, date_min, date_max, obj_event, obj_id, obj_type, utc_offset, metada
         if user == "CUSTOMER":
             username = user_data.get(log['userId'])
             if username is None:
-                username = user_mgr.get_user(log['userId'], "mask[username]")['username']
-                user_data[log['userId']] = username
+                try:
+                    username = user_mgr.get_user(log['userId'], "mask[username]")['username']
+                    user_data[log['userId']] = username
+                except ValueError as ex:
+                    raise exceptions.ArgumentError("You do not have permission to access this user.  %s" % ex)
```

Comment:
The point of this change is so that this command keeps going when were unable to access the user information.
```suggestion
                except exceptions.SoftLayerAPIError as ex:
                    username = log['userId']
```